src/main/java/io/confluent/examples/streams/SessionWindowsExample.java

```diff
@@ -164,7 +164,7 @@ static Topology buildTopology(final Map<String, String> serdeConfig) {
         // group by key so we can count by session windows
         .groupByKey(Grouped.with(Serdes.String(), playEventSerde))
         // window by session
-        .windowedBy(SessionWindows.with(INACTIVITY_GAP))
+        .windowedBy(SessionWindows.ofInactivityGapAndGrace(INACTIVITY_GAP, Duration.ofMillis(100)))
```

Comment:
For this case, we need a grace to make the test pass -- without a grace, store history is limited and cleaned up too quickly, and IQ that is used in the test cannot find the data any longer.